migrations/20150926071544-team-status-migration.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+module.exports = {
+	up: function (migration, DataTypes) {
+		/*
+			Add altering commands here.
+			Return a promise to correctly handle asynchronicity.
+		*/
+
+		migration.createTable('team_statuses', {
+			id: {
+				type: DataTypes.INTEGER.UNSIGNED,
+				primaryKey: true,
+				autoIncrement: true,
+				allowNull: false
+			},
+			team: {
+				type: DataTypes.CHAR,
+				length: 3,
+				allowNull: false
+			},
+			status: {
+				type: DataTypes.CHAR,
+				length: 15,
+				allowNull: false
+			},
+			bought_by: {
+				type: DataTypes.INTEGER.UNSIGNED,
+				allowNull: false,
+				references: 'players',
+				referencesKey: 'id',
+				onUpdate: 'CASCADE',
+				onDelete: 'CASCADE'
+			},
+			expires_at: {
+				type: DataTypes.DATE,
+				allowNull: false
+			},
+			strength: {
+				type: DataTypes.FLOAT.UNSIGNED,
```

Comment:
Do we want to do unsigned? I know we talked about debuffs only but it would be nice to keep the option open.